### PR TITLE
Internal API

### PR DESCRIPTION
**Pull request by ChrisP19**
- New create project endpoint that takes structured data instead of forms
- New create task endpoint that adds tasks to existing projects

### CODE REVIEW

File: scalabel/tests/test_scalabel_api.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
+""" Test file for the scalabel api """
```
Comment (pxnguyen) [inline scalabel/tests/test_scalabel_api.py:1]: Test file usually in the format of "*_test.py".

File: scalabel/tests/test_scalabel_api.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+import subprocess
+import time
+import logging
+import os
+import signal
+import shutil
+import pytest
```
Comment (pxnguyen) [inline scalabel/tests/test_scalabel_api.py:8]: We use unittest instead of pytest.

File: scalabel/tests/test_scalabel_api.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+import yaml
+from urllib3.exceptions import HTTPError
+from ..scalabel_api import ScalabelAPI
```
Comment (pxnguyen) [inline scalabel/tests/test_scalabel_api.py:11]: Use full path,
"from scalabel.scalabel_api import ScalabelAPI"

Add the scalabel to the appropriate python path:
export PYTHONPATH="..."

File: scalabel/tests/test_scalabel_api.py (hunk 1 above)
Comment (ChrisP19) [reply scalabel/tests/test_scalabel_api.py]: Renamed to match the convention

File: scalabel/tests/test_scalabel_api.py (hunk 3 above)
Comment (ChrisP19) [reply scalabel/tests/test_scalabel_api.py]: Changed to full path, but didn't need to modify python path for it to work

**Pull request merged**